test/verify/check-openshift

```diff
@@ -542,15 +542,15 @@ LABEL io.projectatomic.nulecule.atomicappversion="0.1.11" \
         b.click("tr[data-row-id='vm-fedoravm']")  # expand row
         b.wait_present("tr[data-row-id='vm-fedoravm'] + tr.listing-ct-panel")
         b.wait_present("#vm-fedoravm-node")
-        wait(lambda: b.text("#vm-fedoravm-node > a").strip() != "-") # `-` ==  unassigned ; so wait for a change
+        b.wait_js_cond('$("#vm-fedoravm-node > a").text().strip() !== "-"') # `-` ==  unassigned ; so wait for a change
```

Comment:
This is indeed a much more efficient way to wait for a condition, as it doesn't need to keep circling between Python, CDP, and the browser; but in principle it should achieve the same thing. The browser *does* continue to run the page while talking (or not) via CDP to it. Not running the page was a PhantomJS behaviour. Does that correspond to any test flake?